system/settings/settings.xml

```diff
@@ -1082,6 +1082,7 @@
           <level>1</level>
           <default>-</default>
```

Comment:
While you're at it, please remove the "-" from the default value (same for `subtitles.movie`). While it's handled by the settings system it doesn't really make any sense anymore.